packages/components/src/components/FilterMenu.vue

```diff
@@ -85,14 +85,16 @@
               :suggestions="hideNamesSuggestions"
               suggestions-placement="top"
             />
-            <div class="filters__hide" v-if="hiddenNames.length">
-              <div class="filters__hide-item" v-for="(name, index) in hiddenNames" :key="name">
-                {{ name }}
-                <CloseThinIcon
-                  class="filters__hide-item-icon"
-                  @click.stop="removeHiddenName(index)"
-                />
-              </div>
+          </div>
+        </div>
+        <div class="filters__block-row">
+          <div class="filters__hide" v-if="hiddenNames.length">
+            <div class="filters__hide-item" v-for="(name, index) in hiddenNames" :key="name">
+              {{ name }}
```

Comment:
Should these clip with like an ellipsis instead of wrapping? Maybe a tool tip for the full content? 

Please try with very long code object names a la Java. 

One other trick in Java is to shorten the package names to just the first letter of each package.

Reply:
Ok. The problem is that the “Save Filter” content can be pushed out of the viewport and then it can’t be accessed.

Reply:
That's strange, I can't reproduce that problem. I can always access the bottom of the menu using the scrollbar, even if I try extremely small viewports. Do you have steps to reproduce?

Reply:
I think I’ve only seen it on mobile so, not a priority.